src/integration/__tests__/integration.test.ts

```diff
@@ -11,15 +11,21 @@ if (!isOnCI) {
 } else {
   // These tests ensure that *any* query works against our express app.
   const request = require("supertest")
-  const app = require("../../index")
-  const gql = require("lib/gql")
+  const app = require("../../index").default
+  const gql = require("lib/gql").default
 
-  it("It should bail for an unknown GET request", async () => {
+  it("bails for an unknown GET request", async () => {
     const response = await request(app).get("/")
     expect(response.statusCode).toBe(400)
   })
 
-  it("It can make a request against the schema", async () => {
+  // TODO: Unsure why this doesn’t work
+  xit("responds with its health", async () => {
+    const response = await request(app).get("/health")
+    expect(response.statusCode).toBe(200)
+  })
```

Comment:
@orta I had a hard time figuring out why this wasn’t working as I expected it to, do you have any thoughts?

Reply:
@alloy tests pass - is this good to merge?

Reply:
Yeah, this test is disabled atm. I tested locally that I could GET this endpoint and get a 200. 👍